### ISSUE TITLE
Setting Paging & Circular Programatically

### ISSUE THREAD

**Issue by mmackh** [ikhsan/IAInfiniteGridView#3]
I implemented this control line for line and it worked fine with paging and circular turned off. Then I tried to add this:

```
- (void)viewDidAppear:(BOOL)animated
{
    [self.gridView setCircular:YES];
    [self.gridView setPagingEnabled:YES];
    [self.gridView jumpToIndex:0];
}

```

With some very strange results. The paging would work for the first scroll through but after that would gradually offset itself - more and more.

So I don't know what the issue is there, but a workaround is to add/change the initialization method in IAInfiniteGridView.m 

```
- (void)initialization {
    ...
    self.circular = YES;
    self.paging = YES;
    ...
}
```

Maybe someone can shed some light onto what's going on.

**Comment by ikhsan** [comment]
When enabling paging, what you want to set is a custom property called `paging`, not the `pagingEnabled` which it got from `UIScrollView` property. I've tried to make a paging feature by using `pagingEnabled`, but, like as you mention, there's an offset every time the layout recenters, and also the page is actually the size of the view (i.e in my sample code infinite, 3 grids) rather than 1 page per grid. That's why I implemented a simple custom paging mechanism, to fix both of them. 

I've tried using what people suggested, that is making the subviews to not clip, and make the page size to one grid rather than 3 but I have no luck on that. The offset when recenter is still there. So, this is what I've got for the current time being.
